content/guides/printing/overview.adoc

```diff
@@ -0,0 +1,59 @@
+= Overview
+Sameer Rahmani
+2016-09-07
+:type: guides
+:icons: font
+:status: published
+
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+In this series of articles we're going to take a look at different ways of printing in Clojure.
+
+
+== clojure.pprint
+This namespace adds a new feature to Clojure: a generalized pretty printer.
+
+The pretty printer is easy to use:
+
+[source,clojure]
+----
+user=> (println (for [x (range 10)] (range x)))
+(() (0) (0 1) (0 1 2) (0 1 2 3) (0 1 2 3 4) (0 1 2 3 4 5) (0 1 2 3 4 5 6) (0 1 2 3 4 5 6 7) (0 1 2 3 4 5 6 7 8))
+nil
+user=> (use 'clojure.pprint)
```

Comment:
Probably good to mention in the text that pprint is implemented in the clojure.pprint namespace.